bin/varnishd/cache/cache_backend.c

```diff
@@ -60,6 +60,14 @@ enum connwait_e {
 	CW_BE_BUSY,
 };
 
+struct connwait {
+	unsigned			magic;
+#define CONNWAIT_MAGIC			0x75c7a52b
+	enum connwait_e			cw_state;
+	VTAILQ_ENTRY(connwait)		cw_list;
+	pthread_cond_t			cw_cond;
+};
```

Comment:
Didn't we conclude that we could use `bo->wrk->cond` for the connection queue?

Reply:
@Dridi that's not what I understood from the discussion [here](https://github.com/varnishcache/varnish-cache/pull/4030#discussion_r1430199392)
See Daryl's comment:

> The original implementation used, I believe, (struct boc).cond. But there were concerns raised about hijacking that cond for something like this. To alleviate those concerns it was given its own cond, which also helps code clarity (IMO).

Reply:
Right, but this was about the boc cond, not the worker cond, right? It looks like we never discussed the latter.

Reply:
It might have been discussed on the original implementation, not sure, maybe @drodden can provide some input ?

Reply:
I vaguely remember something about not using the worker cond and found this comment on the original PR:

> [daghf] [Jan 16, 2023] 
> Indeed. I also like the idea of not repurposing wrk->cond, this rhymes better with the stated future task of task disembarking.